plugins/modules/hassio_os.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: hassio_host
+author: "marksie1988 / TotalDebug (@marksie1988)"
+short_description: Manage Home Assistant (HassIO) host
+version_added: "2.0.1"
+description:
+  - Manage Home Assistant (HassIO, hass.io) host - update
+options:
+  state:
+    description:
+      - State of host
+    required: true
+    choices: ['updated']
+"""
+
+EXAMPLES = """
+# Update HassIO OS
+- hassio_host:
+    state: update
+    token: <SUPERVISOR_TOKEN>
```

Comment:
documentation needs to be updated.

Reply:
I've updated documenation can you check if you agree?